[2023-06-06T00:24:42.830+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: process_weather_people_postgres.group_users.is_1_api_available scheduled__2023-06-04T00:00:00+00:00 [queued]>
[2023-06-06T00:24:42.843+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: process_weather_people_postgres.group_users.is_1_api_available scheduled__2023-06-04T00:00:00+00:00 [queued]>
[2023-06-06T00:24:42.844+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-06T00:24:42.844+0000] {taskinstance.py:1363} INFO - Starting attempt 17 of 20
[2023-06-06T00:24:42.844+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-06T00:24:42.859+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): group_users.is_1_api_available> on 2023-06-04 00:00:00+00:00
[2023-06-06T00:24:42.869+0000] {standard_task_runner.py:55} INFO - Started process 14624 to run task
[2023-06-06T00:24:42.872+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'process_weather_people_postgres', 'group_users.is_1_api_available', 'scheduled__2023-06-04T00:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/process_weather_people_postgres/process_weather_people_postgres.py', '--cfg-path', '/tmp/tmpoazq3jj3']
[2023-06-06T00:24:42.874+0000] {standard_task_runner.py:83} INFO - Job 152: Subtask group_users.is_1_api_available
[2023-06-06T00:24:42.940+0000] {task_command.py:376} INFO - Running <TaskInstance: process_weather_people_postgres.group_users.is_1_api_available scheduled__2023-06-04T00:00:00+00:00 [running]> on host 1160da0dacdb
[2023-06-06T00:24:43.011+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=data_eng
AIRFLOW_CTX_DAG_ID=process_weather_people_postgres
AIRFLOW_CTX_TASK_ID=group_users.is_1_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-06-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=17
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-04T00:00:00+00:00
[2023-06-06T00:24:43.012+0000] {http.py:120} INFO - Poking: api/
[2023-06-06T00:24:43.022+0000] {base.py:71} INFO - Using connection ID 'random_people_api' for task execution.
[2023-06-06T00:24:43.024+0000] {http.py:148} INFO - Sending 'GET' to url: http://randomuser.me/api/
[2023-06-06T00:24:43.440+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-06-06T00:24:43.487+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=process_weather_people_postgres, task_id=group_users.is_1_api_available, execution_date=20230604T000000, start_date=20230606T002442, end_date=20230606T002443
[2023-06-06T00:24:43.566+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-06T00:24:43.603+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
